traitcuration/static/js/mapping.js

```diff
@@ -3,13 +3,21 @@ This module contains all the funcionality for mapping traits to ontology terms,
 */
 let selectedRowIndex = -1;
 let selectedTableId = "";
-let currentTraitId = -1;
+let currentTraitId = window.location.href.split("/").slice(-2)[0];
```

Comment:
Here it would be nice to add a comment explaining what we're doing and an example (full string and the result of extraction).

But also, I wonder if there are more elegant ways to pass entity attributes from Django to the JavaScript code? Perhaps including the trait ID as an attribute into the page HTML would work?

Reply:
I also think I've seen examples on some websites where they embed a whole JSON with machine-readable information into the page, for the JavaScript to utilise

Reply:
Well what I could do, is pass some variables to the page via the Django context, and then declare some global JavaScript variables withing the html where they are subsequently passed to. That being said, global JS variables are considered a pretty bad practice so I try to avoid them. Or I could declare some setter methods for passing data via a Django template.

Reply:
I think the last approach is the best one, directly interpolating the values into the JavaScript code. Let's go with that